### PR TITLE
Responsiveness

### LINKED ISSUE #433: All: website responsiveness
The entire site NEEDS to be responsive as most/all of our users are going to be on mobile. The big offender here is tables in the Admin view.

### PR DESCRIPTION
**Pull request by LWest001**
Closes #433.

- Theme file created (adminTheme.js) for working with MUI theming in the admin section.
- Use of `<Stack/>` components to break down wide rows of content (like a search bar with a button next to it) into a column at smaller breakpoints.
  - **Wide:**
image
  - **Narrow:**
image
- Removal of some unnecessary columns in the admin tables.
- The Address in an AssignProfile and HomeTable now is a link to the home, enabling removal of Home column in those tables.
image
- AssignTable gracefully handles surveyor links with multiple surveyors. The stack of surveyor links scrolls.
  - **Wide:**
image
  - **Narrow:**
image
- The form-based pages HomeProfile and UserProfile are fixed - `width: 500` changed to `maxWidth: 500`.

Note: I opted not to use onRowClick function in tables that have links to other items - it seems to be impossible to have a link to other content on a row that uses onRowClick. The browser fires the onRowClick function and it overrides the inline link.

### CODE REVIEW

**Comment by thiagobardini** [comment]
Mobile View: 
https://mui.com/material-ui/react-pagination/
https://mui.com/material-ui/react-accordion/

File: frontend/front/src/pages/Admin/assignment/AssignProfile.js [hunk 1]
```diff
@@ -48,18 +48,33 @@ const AssignProfile = () => {
     : "Unassigned";
 
   const handleUserLink = (user) => goToBreadcrumb("user", user);
-  const handleHomeLink = (home) => goToBreadcrumb("home", home);
+  const handleHomeLink = (home) => {
+    console.log(home);
```
Comment (mattdelsordo) [inline frontend/front/src/pages/Admin/assignment/AssignProfile.js:52]: Remove this log

File: frontend/front/src/pages/Admin/assignment/AssignTable.js [hunk 2]
```diff
@@ -204,24 +220,32 @@ const AssignTable = () => {
...
           columns={columns}
           pageSize={20}
           rowsPerPageOptions={[20]}
+          getRowHeight={() => "auto"}
           disableSelectionOnClick
           autoHeight
           checkboxSelection
           onSelectionModelChange={handleSelectionModelChange}
           selectionModel={selectedAssignments}
+          // onRowClick={(params) => handleRowClick(params.row)}
```
Comment (mattdelsordo) [inline frontend/front/src/pages/Admin/assignment/AssignTable.js:248]: You can remove this if onRowClick isn't being used.

**Review by mattdelsordo — CHANGES_REQUESTED** [review]
Looks pretty good, rebase this off of main to pick up the test fixes

**Review by mattdelsordo — APPROVED** [review]

**Comment by LWest001** [comment]
This branch is just not passing the prettier checks. I'm going to merge it so that we can add those prettier changes in another branch because there are so many of them.

**Pull request merged**